collectd/collectd.go

```diff
@@ -0,0 +1,216 @@
+package collectd
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"net"
+	"sync"
+	"time"
+
+	"github.com/kimor79/gollectd"
+)
+
+// DefaultPort for colletd is 25826
```

Comment:
We use it in our config package for a default.  I would argue that if they don't set the port, and I see it's zero when starting up the ListenAndServer, I could set it then, and it could be a private const then.  Thoughts?